src/libDirectoryService/DSBlockPreProcessing.cpp

```diff
@@ -717,16 +717,9 @@ bool DirectoryService::VerifyNodePriority(const DequeOfShard& shards,
     for (const auto& shardNode : shard) {
       const PubKey& toFind = std::get<SHARD_NODE_PUBKEY>(shardNode);
       if (setTopPriorityNodes.find(toFind) == setTopPriorityNodes.end()) {
-        auto reputation = m_mapNodeReputation.find(toFind);
```

Comment:
What's the main reason for removing the check here to allow for other priority nodes proposed by leader?

Reply:
Initially mistaken that the check is similar in FindTopPriorityNodes, added back now